contracts/vaults/base/RibbonVault.sol

```diff
@@ -347,15 +347,20 @@ contract RibbonVault is OptionsVaultStorage {
     /**
      * @notice Withdraws the assets on the vault using the outstanding `DepositReceipt.amount`
      * @param amount is the amount to withdraw
+     * @param isThetaVault is whether the vault is a theta vault
      */
-    function withdrawInstantly(uint256 amount) external nonReentrant {
+    function _withdrawInstantly(uint256 amount, bool isThetaVault) internal {
         Vault.DepositReceipt storage depositReceipt =
             depositReceipts[msg.sender];
 
         uint16 currentRound = vaultState.round;
         require(amount > 0, "!amount");
-        require(!depositReceipt.processed, "Processed");
-        require(depositReceipt.round == currentRound, "Invalid round");
+
+        if (isThetaVault) {
+            require(!depositReceipt.processed, "Processed");
+            require(depositReceipt.round == currentRound, "Invalid round");
+        }
```

Comment:
What's the rationale behind doing these checks only for Theta Vault? Isn't the withdraw instantly completely different for Delta Vault? Like we don't even have to concern ourselves with deposit receipt stuff

Reply:
isnt deposit receipt still relevant for how deposits work? withdrawals I agree.